### PR TITLE
Deploy

### PR DESCRIPTION
**Pull request by nzoschke**
```bash
$ convox login convox-charlie-935967921.us-east-1.elb.amazonaws.com
Password: 
Login Succeeded
EXTRA_ARGS="--insecure-registry convox-charlie-935967921.us-east-1.elb.amazonaws.com:5000"
Need TLS certs for boot2docker,127.0.0.1,10.0.2.15,192.168.59.103
-------------------
WARNING: login credentials saved in /Users/noah/.docker/config.json
Login Succeeded

$ cd myapp
$ convox deploy
Docker Compose app detected.
web uses an image, skipping
latest: Pulling from httpd

64e5325c0d9d: Pull complete 
bf84c1d84a8f: Pull complete 
6c1a7f5286ab: Pull complete 
248bbb739a15: Pull complete 
5e5a25bf7039: Pull complete 
183a736359e7: Pull complete 
41f5cdce27f3: Pull complete 
9155aba0344d: Pull complete 
06deff27c05b: Pull complete 
2b1597dc71ff: Pull complete 
5c405ca8c94d: Pull complete 
1fb661afda9f: Pull complete 
ee4d515e8896: Pull complete 
de94ed779434: Already exists 
httpd:latest: The image you are pulling has been verified. Important: image verification is a tech preview feature and should not be relied on to provide security.

Digest: sha256:d1cd248b8a813615ee6adaf3a0ae9d6d6c0a3c836272d5e67844dd608d4bb04d
Status: Downloaded newer image for httpd:latest
Tagging httpd
Pushing convox-delta-424363854.us-east-1.elb.amazonaws.com:5000/myapp6-web:1435598703
The push refers to a repository [convox-delta-424363854.us-east-1.elb.amazonaws.com:5000/myapp6-web] (len: 1)
Sending image list
Pushing repository convox-delta-424363854.us-east-1.elb.amazonaws.com:5000/myapp6-web (1 tags)
Image 6c1a7f5286ab already pushed, skipping
Image 248bbb739a15 already pushed, skipping
Image 64e5325c0d9d already pushed, skipping
Image 5e5a25bf7039 already pushed, skipping
Image bf84c1d84a8f already pushed, skipping
Image 06deff27c05b already pushed, skipping
Image 9155aba0344d already pushed, skipping
Image 2b1597dc71ff already pushed, skipping
Image 41f5cdce27f3 already pushed, skipping
Image 183a736359e7 already pushed, skipping
Image 1fb661afda9f already pushed, skipping
Image ee4d515e8896 already pushed, skipping
Image 5c405ca8c94d already pushed, skipping
Image de94ed779434 already pushed, skipping
Pushing tag for rev [de94ed779434] on {https://convox-delta-424363854.us-east-1.elb.amazonaws.com:5000/v1/repositories/myapp6-web/tags/1435598703}
Created app myapp6
Status running
Created release 1435598703
Status running
```

### CODE REVIEW

File: convox/deploy.go [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+		host = os.Getenv("REGISTRY_HOST")
+	}
+
+	proj := strings.Replace(filepath.Base(base), "-", "", -1)
+	images := m.ImageNames(proj)
+	tag := fmt.Sprintf("%v", stdcli.Tagger())
+	tags := m.TagNames(host, proj, tag)
+
+	for i := 0; i < len(images); i++ {
+		fmt.Printf("tag %s %s\n", images[i], tags[i])
```
Comment (ddollar) [inline convox/deploy.go]: Some debug code left over

Comment (nzoschke) [inline convox/deploy.go]: This is intentional feedback for a user.

**Comment by ddollar** [comment]
:+1:

File: convox/deploy.go (hunk 1 above)
Comment (ddollar) [inline convox/deploy.go]: Testing

**Comment by nzoschke** [comment]
Failing tests are something with the go environment on CircleCI

**Pull request merged**